vllm_detector_adapter/protocol.py

```diff
@@ -82,7 +82,7 @@ def from_chat_completion_response(
             # NOTE: for providing spans, we currently consider entire generated text as a span.
             # This is because, at the time of writing, the generative guardrail models does not
             # provide specific information about input text, which can be used to deduce spans.
-            if content and isinstance(content, str):
+            if content and isinstance(content, str) and len(content.strip()) > 0:
```

Comment:
can `len(content.strip()) > 0:` be replaced to `content.strip()`, since empty string is falsy